projects/ion/src/lib/info-badge/info-badge.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { CommonModule } from '@angular/common';
+import { render, screen } from '@testing-library/angular';
+import { IonIconComponent } from '../icon/icon.component';
+
+import { InfoBadgeComponent } from './info-badge.component';
+import { InfoBadgeProps } from './info-badge.component';
+
+const defaultInfoBadge: InfoBadgeProps = {
+  variant: 'primary',
+};
+
+const sut = async (customProps: InfoBadgeProps = defaultInfoBadge) => {
+  await render(InfoBadgeComponent, {
+    componentProperties: customProps,
+    imports: [CommonModule],
+    declarations: [IonIconComponent],
+  });
+};
+
+describe('InfoBadgeComponent', () => {
+  it('Should render an empty info badge', async () => {
+    await sut({ ...defaultInfoBadge });
+    expect(screen.getByTestId('info-badge')).toBeInTheDocument();
+  });
+
+  it('Should render an info badge with icon', async () => {
+    const icon = 'check';
+    await sut({ icon, ...defaultInfoBadge });
+
+    expect(screen.getByTestId('info-badge-icon')).toBeInTheDocument();
+  });
+
+  it('Should render an info badge with text', async () => {
+    const text = 'Lorem ipsum';
+    await sut({ text, ...defaultInfoBadge });
+
+    expect(screen.getByText(text)).toBeInTheDocument();
+  });
+
+  it.each(['primary', 'success', 'info', 'warning', 'negative'])(
+    'Should render an info badge for each status',
+    async () => {
+      const icon = 'check';
+      await sut({ icon, ...defaultInfoBadge });
+
+      expect(screen.getByTestId('info-badge-icon')).toBeInTheDocument();
+    }
```

Comment:
check correct class.

Reply:
add expect with .toHaveClass(class) where class is a parameter of callback.

Reply:
See doc https://jestjs.io/pt-BR/docs/api#testeachtablename-fn-timeout

Reply:
Done